src/internt-vedtak/typer/interntVedtak.ts

```diff
@@ -11,6 +12,7 @@ export interface InterntVedtak {
     målgrupper: Vilkårperiode[];
     aktiviteter: Vilkårperiode[];
     stønadsperioder: Stønadsperiode[];
+    vedtaksperioder: Vedtaksperiode[];
```

Comment:
Se 
* https://github.com/navikt/tilleggsstonader-sak/pull/595/files#r1961118473

Burde man fjerne at man viser stønadsperioder her?

Reply:
Det kan vi gjøre. (439a797e0b9821381db3641e68e52ac49f9f99f9) 

Foreslår vi går for denne fremgangsmåten: https://github.com/navikt/tilleggsstonader-sak/pull/595#discussion_r1961632318